cmd/image-builder/main.go

```diff
@@ -134,6 +135,10 @@ func main() {
 	if conf.GlitchTipDSN != "" {
 		echoServer.Use(sentryecho.New(sentryecho.Options{}))
 	}
+	// log stack traces into standard logger as error (instead of stdout)
+	echoServer.Use(middleware.RecoverWithConfig(middleware.RecoverConfig{
+		LogLevel: log.ERROR,
+	}))
```

Comment:
Can you please split this into a separate commit?

Reply:
Sure, done.